docs/user-guide/archived-items.md

```diff
@@ -22,19 +22,37 @@ Because <span class="status-warning">:bootstrap-x-octagon-fill: Canceled</span>
 
 ## Archived Item Details
 
-The archived item details page is composed of five sections, though the Crawl Settings tab is only available for crawls and not uploads.
+The archived item details page is composed of six sections, though some tabs are only available for crawls and not uploads.
```

Comment:
Will viewers see only some of these sections? Might be worth making a note of this if so

Reply:
QA page is hidden from viewers???  Not really sure why?  It currently has the pages list which is presumably useful for them to some extent, though all the actions on that page should be hidden from viewers.

Reply:
Well the pages list and rankings should ideally be viewable for viewers, but fair enough, not a high priority issue (IDK if we even have a single user with the viewer role on prod haha)

Perhaps something to fix in the next iterations?  I like the idea of a QA role!  Will update with a note.

Reply:
What's the reason for needing to specify the number of sections? IMO specifying the number makes the count feel more important than it is, especially when it's dependent on whether QA is enabled, what the archived item type is, and maybe other factors in the future. How about something like:
```
Archived item details are organized into the following sections.
```
And then "(crawl-only)" next to the heading for each applicable subsection that describes the tab content.